src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -557,7 +557,7 @@ void GIVEN_null_log_message_WHEN_upload_attempted_THEN_null_message_considered_a
         assertTrue(file.setReadable(true));
         assertTrue(file.setWritable(true));
         try (OutputStream fileOutputStream = Files.newOutputStream(file.toPath())) {
-            fileOutputStream.write("null".getBytes(StandardCharsets.UTF_8));
+            fileOutputStream.write("null\n".getBytes(StandardCharsets.UTF_8));
```

Comment:
The purpose of this unit test is handling the null message, however the new mechanism will only consider the null message with a line separator.